projects/ngrx.io/content/guide/store/recipes/custom-elements.md

```diff
@@ -0,0 +1,88 @@
+# Using Store with Angular Elements
+## Creating Angular Elements
+The following recipe shows you how to manage the state of a counter, and how to select and display it within an Angular Elements and use the same within an Angular Project and a static page.
+## Recipe
+You can download  <live-example name="store-elements" downloadOnly>Angular Elements Project</live-example>.
```

Comment:
Is this an alternative to following the below instructions? I'm not sure this adds any value. Might want to consider removing this as it's not clear what the reader should be doing.

Reply:
This is the entire example code, which can be downloaded.

Reply:
If this is the entire code sample, then I would recommend creating a new section at the very end for readers to download the example code.

Reply:
makes sense.

Reply:
Done.

Reply:
OK Let's take it a little further. 

Instead of just adding `You can download Angular Elements Project.` Let's create a new section with a heading called `Project Example Repository` and add a sentence about cloning the full example from GitHub with a link to the [GitHub repo](https://github.com/santoshyadav198613/platform/tree/docs--add-receipe-to-use-store-with-elelments/projects/ngrx.io/content/examples/store-elements). 

This adds more value to the reader as they can peruse the full solution on GitHub and clone for themselves to review.

Reply:
Ok. let me do that, it should be at the end right.

Reply:
But won't that be an overhead for users as they have to download the entire repo? 
By download link, we are downloading the same app, and they will get the only code they want.